src/main/java/Assignment1.java

```diff
@@ -33,139 +50,115 @@ public static void main(String[] args) {
     // check if type is present as it is mandatory
     if (!item.containsKey("-type")) {
       // send not there message and quit
-      System.out.println("-type argument not provided. Exiting...");
-      System.exit(1);
+      throw new InvalidArguementsException("-name should be present first.");
     }
     // if price is not present, assign a default value
     if (!item.containsKey("-price")) {
+      System.out.println("-price wasn't provided, using 0 as default value...");
```

Comment:
validation should happen in separate class which will perform input validation of the user.